pkg/workload/admissionchecks_test.go

```diff
@@ -222,6 +253,10 @@ func TestSyncAdmittedCondition(t *testing.T) {
 			if diff := cmp.Diff(tc.wantConditions, wl.Status.Conditions, cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime", "Message")); diff != "" {
 				t.Errorf("Unexpected conditions after sync (- want/+ got):\n%s", diff)
 			}
+
+			if diff := cmp.Diff(tc.wantAdmittedTime, ptr.Deref(wl.Status.AccumulatedPastExexcutionTimeSeconds, -1)); diff != "" {
```

Comment:
```suggestion
			if diff := cmp.Diff(tc.wantAdmittedTime, wl.Status.AccumulatedPastExexcutionTimeSeconds); diff != "" {
```

Couldn't we just compare those values? I think we can make `wantAdmittedTime` as a pointer.

Reply:
> Let's drop the ptr.Deref and do the check in two steps.

That makes sense.